ninja-core/src/main/java/ninja/utils/NinjaConstant.java

```diff
@@ -143,7 +143,35 @@
      * (XSS).
      */
     final String sessionHttpOnly = "application.session.http_only";
-    
+
+    /**
+     * Indicates if uploaded files should be handled fully in-memory without
+     * saving to file system. Can be {@code true} or {@code false}, defaults to
+     * {@code false}.
+     */
+    final String FILE_UPLOADS_IN_MEMORY = "file.uploads.in_memory";
+
+    /**
+     * The maximum allowed size of a single uploaded file.
+     *
+     * @see org.apache.commons.fileupload.FileUploadBase#fileSizeMax
+     */
+    final String FILE_UPLOADS_MAX_FILE_SIZE = "file.uploads.file.size.max";
```

Comment:
Do you think we could simplify that and only use "FILE_UPLOADS_MAX_REQUEST_SIZE". It somewhat also includes the max files size and we got one property less :)